resources/benchmark-runner.mjs

```diff
@@ -29,38 +26,33 @@ class Page {
         });
     }
 
-    querySelector(selector)
-    {
+    querySelector(selector) {
         const element = this._frame.contentDocument.querySelector(selector);
-        if (element === null)
-            return null;
+        if (element === null) return null;
         return this._wrapElement(element);
     }
 
-    querySelectorAll(selector)
-    {
-        const elements = Array.from(this._frame.contentDocument.querySelectorAll(selector));
+    querySelectorAll(selector) {
```

Comment:
+1 to @rniwa's comment earlier. We'll want to make sure curly braces are in a new line to adhere to https://webkit.org/code-style-guidelines/